### DISCUSSION TITLE
Is there any way to use only Major.Minor.Path.Height?

### DISCUSSION THREAD

**Discussion by pkuehnel** [adamralph/minver#792] [Q&A] [answered]
Currently my product version gets tagged like: `2.3.4-alpha.0.2` and my file version would be `2.3.4.0`. Is there any way to let the file- and productversion be 2.3.4.2 where 2 means there were two commits since my v2.3.4 tag?

I really would need a detailed instruction as I am absolutly not into MS build.
To be honest, I don't understand why changing anything in the `.\obj\*csproj.nuget.g.targets` does result in changes of the behaviour of minver and what best practice is to change these things permanently as the obj directory is ignored by git by default.

**Comment by pkuehnel** [comment]
Ok meanwhile I resolved the first part of my question, just use:
```
<PackageVersion>$(MinVerMajor).$(MinVerMinor).$(MinVerPatch).$(MinVerVersion.Split(`.`)[4])</PackageVersion>
<Version>$(MinVerMajor).$(MinVerMinor).$(MinVerPatch).$(MinVerVersion.Split(`.`)[4])</Version>
```

But still: Why is this `*.targets` file even used, an what is best practice to get that configuration into my git repo?

**Comment by bording** [comment, marked as answer]
> what is best practice to get that configuration into my git repo?

The way to do this is already specified in the FAQ: https://github.com/adamralph/minver#can-i-use-the-version-calculated-by-minver-for-other-purposes

You would put that target with your desired changes in your project file, or in a separate targets file that you could reference from multiple project files if your repo has multiple projects that need to be versioned.
